ArticleTemplates/assets/js/modules/more-tags.js

```diff
@@ -14,7 +14,7 @@ define(function () {
             initialised = true;
             tags = document.querySelectorAll('.tags .inline-list .inline-list__item');
```

Comment:
we could actually simplify this selector now to look up any element with the new class name 'tags__list-item'